src/style.css

```diff
@@ -1 +1,140 @@
-/* add styles here */
+/* #region fonts */
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+/* #endregion fonts */
+
+/* #region normalize */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+/* #endregion normalize */
+
+/* #region page */
+.page {
+  font-weight: 300;
+}
+
+.page__body {
+  padding: 20px 8px 0;
+}
+
+/* #endregion page */
+
+/* #region container */
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  width: 100%;
+  padding: 0;
+}
+
+/* #endregion container */
+
+/* #region form */
+.form {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  position: relative;
+}
+
+.form__search-button-big {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+.form__search-button-small {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+}
+
+/* #endregion form */
+
+/* #region icon-search */
+.icon-search {
+  margin: 0;
+  padding: 0;
+  border: none;
+  background-color: transparent;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
+  z-index: 1;
+}
+
+.icon-search--size--big {
+  width: 19px;
+  height: 19px;
+}
+
+.icon-search--size--small {
+  width: 11px;
+  height: 11px;
+}
+
```

Comment:
This class seems to be missing the naming convention of BEM; perhaps you meant to use `--size-big` and `--size-small` as the modifiers? (Note the amount of `-`s)

Reply:
Hello EdPirro, I use this convention (https://en.bem.info/methodology/naming-convention/#two-dashes-style) recommended in theory in the Html + css advanced/BEM lesson. I actually have doubts about this use. I change my code to only one hyphen.